spec/listformat.html

```diff
@@ -196,13 +196,31 @@ <h1>Intl.ListFormat ( [ _locales_ [ , _options_ ] ] )</h1>
         1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
         1. Let _dataLocaleTypes_ be _dataLocaleData_.[[&lt;_type_&gt;]].
         1. Let _templates_ be _dataLocaleTypes_.[[&lt;_style_&gt;]].
+        1. Call VerifyListPattern(_templates_.[[Pair]]).
```

Comment:
This assertion seems redundant given what you added below. To me, it makes more sense to include these assertions as part of the definition of the locale database, rather than when running algorithms.

(Nit: s/Call/Perform/)

Reply:
Right.  We're in a spec, not in code. :)